pkg/e2e/verify/samesite_cookie.go

```diff
@@ -24,13 +27,30 @@ var _ = ginkgo.Describe(samesiteTestName, func() {
 	h := helper.New()
 
 	ginkgo.Context("Validating samesite cookie", func() {
+
 		ginkgo.It("should be set for openshift-monitoring OSD managed routes", func() {
+			clusterVersion, majMinVersion, err := getClusterVersion(h)
+			Expect(err).NotTo(HaveOccurred(), "failed getting cluster version")
+			Expect(clusterVersion).NotTo(BeNil())
+
+			if majMinVersion < supportVersion {
+				ginkgo.Skip("skipping due to unsupported cluster version. Must be >=4.6.0")
+			}
+
```

Comment:
Since this exact same code block is repeated twice, would it be possible to refactor it into a single function. Might work well as a closure defined above the `ginkgo.It` calls...

Reply:
Yeah, I originally attempted to create a function and ginkgo was skipping the second test for some reason. Closures are new to me, I will check that out and test.